tests/queries/tests.py

```diff
@@ -3825,3 +3825,17 @@ def test_ticket_23622(self):
             Ticket23605A.objects.filter(qx),
             [a2], lambda x: x
         )
+
+class Ticket26983Tests(TestCase):
```

Comment:
Sure. The other issue is that the other test to do with this ticket is with the ToField tests. Merge the two, or simply place this one somewhere better?